libraries/AP_Logger/LogStructure.h

```diff
@@ -2503,6 +2503,24 @@ struct PACKED log_Arm_Disarm {
       "ERR",   "QBB",         "TimeUS,Subsys,ECode", "s--", "F--" }
 
 
+// @LoggerMessage: SBPH
+// @Description: Swift Health Data
+// @Field: TimeUS: Time since system startup
+// @Field: CrcError: Number of packet CRC errors on serial connection
+// @Field: LastInject: Timestamp of last raw data injection to GPS
+// @Field: IARhyp: Current number of integer ambiguity hypotheses
+
+// @LoggerMessage: SBRH
+// @Description: Swift Raw Message Data
+// @Field: TimeUS: Time since system startup
+// @Field: msg_flag: Swift message type
+// @Field: 1: Sender ID
+// @Field: 2: index; always 1
```

Comment:
Not terrible but we don't normally use semicolons in descriptions.  commands, brackets.. not a blocker perhaps.